### DISCUSSION TITLE
Consumer Is Stuck On "Waiting Read Op"

### DISCUSSION THREAD

**Discussion by shashank-sachan** [apache/pulsar#21373] [Q&A]
Hi,

We have a Topic where we have subscribed through multiple consumers of same service (this is service is running on K8S with multiple pod)
We see that one of the consumer is set to false  for Waiting Read Op

![Screenshot 2023-10-17 at 7 56 09 AM](https://github.com/apache/pulsar/assets/46921852/bed24a42-c92e-4236-ab83-1af8f24f143c)

Is there anyway to identity this issue and why it's stuck here?

Thanks,

**Comment by michalcukierman** [comment]
You need to get topic stats or topics stats internal:
https://pulsar.apache.org/reference/#/3.0.x/pulsar-admin/topics

It may be a bug, or all the messages were read, but some were not acknowledged (as the backlog is the list of messages without ACK: https://pulsar.apache.org/docs/3.1.x/cookbooks-retention-expiry/).
